snekbox/nsjail.py

```diff
@@ -133,11 +136,33 @@ def python3(self, code: str) -> CompletedProcess:
             except ValueError:
                 return CompletedProcess(args, None, "ValueError: embedded null byte", None)
 
+            output_size = 0
+            output = []
+
+            # We'll consume STDOUT as long as the subprocess is running
+            while nsjail.poll() is None:
+                # Read 100 characters from the STDOUT stream
```

Comment:
It is not accurate to call them "characters" since UTF-8 allows for a single character to be up to 4 bytes long.

Reply:
This is a stream in text mode; the size parameter refers to the number of characters in that case, not the number of bytes.